DeleteReportedEmail (script) - Improve Error Handling (#26363)
demisto/content: Packs/Phishing/Scripts/DeleteReportedEmail/README.md

> A script for deleting reported phishing emails from the mailbox in which they were reported in.

Reviewer: ```suggestion
A script for deleting reported phishing emails from the mailbox in which they were reported.
```

| DeleteReportedEmail.using_brand | The email service that was used to delete the email. | String |
| DeleteReportedEmail.email_subject | The subject of the deleted email. | String |
| DeleteReportedEmail.message_id | The message ID of the deleted email. | String |

## Troubleshooting
---
* If the `Reported Email Origin` field is missing or has a value of `None`, the script will not be able to locate the email and fail.  

>   This can happen if the mail to was forwarded to the listener mailbox was not forwarded as an attachment (with an `EML` file) like it should.

Reviewer: ```suggestion
  This can happen if the email forwarded to the listener mailbox was not forwarded as an attachment (with an `EML` file) as it should.
```

* If either the `Reported Email Message ID` or `Reported Email To` fields are missing, the cause is likely to be one of the following:
  * An `EML` file was not attached to the email.

>   * The playbook is being used as a sub-playbook, causing the `EML` file to exist only in the parent incident.

Reviewer 2: ```suggestion
  * The playbook is being used as a sub-playbook, causing the `EML` file to exist only in the parent playbook.
```
